avro-builder/tests/codegen-14/build.gradle

```diff
@@ -34,7 +34,8 @@ task runOwnCodegen {
       main = 'com.linkedin.avroutil1.builder.SchemaBuilder'
       args = [
           "--input", "$projectDir/src/main/avro",
-          "--output", "$buildDir/generated/sources/avro/java/main"
+          "--output", "$buildDir/generated/sources/avro/java/main",
+          "--generator", "AVRO_UTIL"
```

Comment:
why change just avro 1.4 codegen to use `--generator=AVRO_UTIL`? what about the other versions?

Reply:
We only have avro1.4 in the test module for now